examples/export.html

```diff
@@ -0,0 +1,151 @@
+        img2 = L.distortableImageOverlay('example.jpg', {
+          corners: [
+            L.latLng(51.51, -0.20),
+            L.latLng(51.51,-0.16),
+            L.latLng(51.49, -0.21),
+            L.latLng(51.49,-0.17)
+          ],
+          mode: 'freeRotate',
+          suppressToolbar: true,
+        });
+
+/*
+          function _fetchStatusUrl(collection, scale) {
+            opts.handleStatusUrl = opts.handleStatusUrl || _defaultHandleStatusUrl;
+    
+            $.ajax({
+              url: '//export.mapknitter.org/export',
+              crossDomain: true,
+              type: 'POST',
+              data: {
+                collection: JSON.stringify(collection.images),
+                scale: scale,
+              },
+              success: opts.handleStatusUrl, // this handles the initial response
+            });
+          }
+
+          // receives the URL of status.json, and starts running the updater to repeatedly fetch from status.json;
+          // this may be overridden to integrate with any UI
+          // eslint-disable-next-line require-jsdoc
+          function handleStatusResponse(data) {
+            console.log(data);
+            statusUrl = '//export.mapknitter.org' + data;
+            opts.updater = opts.updater || _defaultUpdater;
+    
+            // repeatedly fetch the status.json
+            updateInterval = setInterval(function intervalUpdater() {
+              $.ajax(statusUrl + '?' + Date.now(), {// bust cache with timestamp
+                type: 'GET',
+                crossDomain: true,
+              }).done(function(data) {
+                opts.updater(data);
+              });
+            }, opts.frequency);
+          }
+ */   
+
+        var json = [{"nodes":[
+                {"lat":"41.8200378187","lon":"-71.4034409085"},
+                {"lat":"41.8199873593","lon":"-71.4030021564"},
+                {"lat":"41.8196229772","lon":"-71.4029728831"},
+                {"lat":"41.8198214546","lon":"-71.4034614433"}
+            ],
+            "cm_per_pixel":23.0934,
+            "src":"https://s3.amazonaws.com/grassrootsmapping/warpables/312455/test.png"},
+            {"nodes":[
+                {"lat":"41.819898342","lon":"-71.4035387139"},
+                {"lat":"41.819898342","lon":"-71.4028493862"},
+                {"lat":"41.8195005594","lon":"-71.4028493862"},
+                {"lat":"41.8195005594","lon":"-71.4035387139"}
+            ],
+            "cm_per_pixel":35.8578,
+            "src":"https://s3.amazonaws.com/grassrootsmapping/warpables/320983/test.png"}
+        ];
+
+        // initialize the collection:
+        imgGroup = L.distortableCollection({
+          collection: json,
+// this needs to be a function run at the moment of pressing, not when initializing the collection
+//          scale: prompt("Choose a scale or use the default (cm per pixel):", 100),
+          exportUrl: 'http://34.74.118.242/api/v2/export/', // used to 
```

Comment:
Here I'm showing one reason WHY we want to be able to customize; we have alternative export servers with slightly different requirements, which we'd like to be able to connect to the Exporter in a simple standardized way.